src/platform/site-wide/announcements/components/VAPlusVetsModal.jsx

```diff
@@ -3,11 +3,7 @@ import Modal from '@department-of-veterans-affairs/formation-react/Modal';
 
 export default class VAPlusVetsModal extends React.Component {
   static isEnabled() {
-    const wasRedirectedFromVets = /^https:\/\/((dev|staging|www)\.)?vets\.gov/.test(
-      document.referrer,
-    );
-
-    return wasRedirectedFromVets;
+    return /^https:\/\/((dev|staging|www)\.)?vets\.gov/.test(document.referrer);
```

Comment:
This code isn't used anymore anyway but I liked having the `wasRedirectedFromVets` help document what it was testing.